### PR TITLE
Remove .png symlinks

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
.png icons are hardcoded and shouldn't be symlinked. The icons must be fixed manually or using hardcode fixer instead.

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -5847,12 +5847,9 @@
             ]
         }
     },
-    "grsync.png": {
```
Comment (Foggalong) [inline data.json:5850]: Because this is a root icon, you'll also have to switch the root icon name around too (for Circle and Square)

**Review by Foggalong — CHANGES_REQUESTED** [review]

File: data.json (hunk 1 above)
Comment (bilelmoussaoui) [reply data.json:5850]: Forgot that! i will do in few minutes

**Comment by bilelmoussaoui** [comment]
Done! For square, there was a png and an svg icon, so i just removed the png file. Should be ready to go

File: icons/square/48/grsync.png.svg [hunk 2]
```diff
@@ -1 +0,0 @@
...9445 --><g><path d="m 23.988 9 c -8.262 0 -14.988 6.699 -14.988 14.984 c 0 8.289 6.727 15.01 14.988 15.01 c 8.262 0 15.01 -6.723 15.01 -15.01 c 0 -8.289 -6.75 -14.984 -15.01 -14.984 m 0 2 c 7.211 0 13.02 5.809 13.02 12.984 c 0 7.176 -5.813 13.02 -13.02 13.02 c -7.211 0 -12.996 -5.84 -12.996 -13.02 c 0 -7.176 5.781 -12.984 12.996 -12.984 m 0 0" fill="#f9f9f9" stroke="none" fill-rule="nonzero" fill-opacity="0.502"/><path d="m 23.988 9 c -8.262 0 -14.988 6.699 -14.988 14.984 l 1.992 0 c 0 -7.176 5.781 -12.984 12.996 -12.984 c 7.211 0 13.02 5.809 13.02 12.984 l 0 0.137 c -0.078 7.125 -5.859 12.879 -13.02 12.879 l 0 1.996 c 8.262 0 15.01 -6.723 15.01 -15.01 c 0 -8.289 -6.75 -14.977 -15.01 -14.977 m 0 0" fill="#f9f9f9" stroke="none" fill-rule="nonzero" fill-opacity="1"/><path d="m 9.563 20 c -0.352 1.273 -0.563 2.617 -0.563 4 c 0 1.727 0.305 3.395 0.844 4.938 l 0.156 0.063 l 6 -9 m -6.438 0" fill="#f9f9f9" stroke="none" fill-rule="nonzero" fill-opacity="1"/></g></g></g></g><g><g transform="translate(0,-1004.3622)"><path d="m 1 1043.36 0 4 c 0 2.216 1.784 4 4 4 l 38 0 c 2.216 0 4 -1.784 4 -4 l 0 -4 c 0 2.216 -1.784 4 -4 4 l -38 0 c -2.216 0 -4 -1.784 -4 -4 z" opacity="0.1"/></g></g></svg>
```
Comment (Foggalong) [inline icons/square/48/grsync.png.svg:1]: You deleted the wrong file! The one left uses the Square legacy design. I'm not sure why there was a discrepancy between the two though

**Comment by Foggalong** [comment]
@bil-elmoussaoui?

**Comment by bilelmoussaoui** [comment]
Which ones should i keep https://github.com/numixproject/numix-core/blob/master/icons/square/48/grsync.svg or https://github.com/numixproject/numix-core/blob/master/icons/square/48/grsync.png.svg?

**Comment by Foggalong** [comment]
[This one](https://github.com/numixproject/numix-core/blob/master/icons/square/48/grsync.png.svg) but change the name to remove the extension. It doesn't display properly in some environments but it's at least using the up to date design which is preferable.

**Comment by bilelmoussaoui** [comment]
Done

**Review by Foggalong — APPROVED** [review]

**Pull request merged**